vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/impl/PackagePropertiesImpl.java

```diff
@@ -211,22 +216,34 @@ public boolean requiresRestart() {
             return dependenciesLocations;
         }
     }
-    
+
     /**
      * {@inheritDoc}
      */
     @Override
     public Calendar getDateProperty(String name) {
+        Calendar result = null;
         try {
             // TODO: add timezone if not there?
             String p = getProperty(name);
-            return p == null
-                    ? null
-                    : ISO8601.parse(p);
+            if (p != null) {
+                result = ISO8601.parse(p);
+                if (result == null) {
+                    // Perhaps this is due to unusual timezone format +02 or +02:00 that isn't supported by ISO8601.parse
```

Comment:
the comment is confusing, isn't the pattern only supporting +02 or 0200 (without colon)?

Reply:
Right, I confused that. Thanks!